<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>IP do servidor:</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QTextEdit" name="textEditIP"/>
                </item>
                <item>
                 <widget class="QListView" name="listView"/>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QPushButton" name="pushButtonConectar">
                    <property name="text">
                     <string>Conectar</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonDesconectar">
                    <property name="text">
                     <string>Desconectar</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonAtualizar">
              <property name="text">
               <string>Atualizar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Intervalo de tempo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSliderConsumidor">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButtonIniciar">
          <property name="text">
           <string>Iniciar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonParar">
          <property name="text">
           <string>Parar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="Grafico" name="grafico" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Grafico</class>
   <extends>QWidget</extends>
   <header>grafico.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderConsumidor</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>label_3</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonConectar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>conectar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDesconectar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>desconectar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonParar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>parar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonIniciar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>iniciar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>conectar()</slot>
  <slot>desconectar()</slot>
  <slot>parar()</slot>
  <slot>iniciar()</slot>
 </slots>
</ui>
